bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/editor/EphemerisElementEditor.scala

```diff
@@ -41,22 +43,14 @@ class EphemerisElementEditor(e: EphemerisElement) extends StdModalEditor[Ephemer
     setValue(new Date(e.validAt))
   }
   object RA extends RATextField(e.coords.ra.toAngle.toDegrees)
-  object Dec extends DecTextField(e.coords.dec.toDegrees)
+  object Dec extends DecTextField(e.coords.dec)
```

Comment:
Just out of curiosity, why does the `DecTextField` take `e.coords.dec`, whereas the `RATextField` needs the `toAngle.toDegrees` conversion done?